packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -295,12 +291,13 @@ export const BlockSwitcher = ( { clientIds } ) => {
 					className="block-editor-block-switcher__no-switcher-icon"
 					title={ blockSwitcherLabel }
```

Comment:
This existing `title` implementation can show redundant tooltips not only in `showIconLabels` mode, but also for patterns. I will address this separately.